### PR TITLE
Creating Games Works

### PR DESCRIPTION
**Pull request by Juicestus**
Creating games works now.

Nothing else.

### CODE REVIEW

File: client/src/firebase.js [hunk 1]
```diff
@@ -98,7 +110,7 @@ export const createNewGame = (hostName) => {
 
 export const joinGame = async (id, name) => {
   const gameRef = doc(db, "games", id);
-  if (gameExists(id)) {
+  if (await gameExists(id)) {
```
Comment (firebanner64) [inline client/src/firebase.js:113]: Is this supposed to be this way?

Comment (Suhas44) [reply client/src/firebase.js:113]: > Is this supposed to be this way?

Yes it’s async

**Review by Suhas44 — APPROVED** [review]
👍

File: client/src/firebase.js (hunk 1 above)
Comment (firebanner64) [reply client/src/firebase.js:113]: That's not what I meant but okay

**Pull request merged**